README.md

```diff
@@ -94,9 +96,31 @@ def user_transaction_counts(transactions):
 
 If a Data Source input to the Feature View has `data_delay` set, then that delay will still be factored in to constructing training data sets but does not impact when the job can be triggered with the materialization API.
 
+Following is an example of definition of a `FeatureTable`
+```python
+from tecton import Entity, FeatureTable
+from tecton.types import Field, String, Int64, Timestamp
+
+from entities import df_user
+import datetime
+
+schema = [
+    Field("name", String),
+    Field("age", Int64),
+    Field("ts", Timestamp)
+]
+
+df_users = FeatureTable(
+    name="df_users", entities=[df_user], schema=schema, online=True, offline=True, ttl=datetime.timedelta(days=30)
+)
+
+``` 
+
 ## Materialization Job Submission
 
-There are two methods available to submit materialization jobs:
+### Batch Feature View Materialization Job Submission
+
+There are two methods available to submit materialization jobs for Batch Feature View:
```

Comment:
s/View/Views
remove Batch, because it also applies for stream fvs

Reply:
done

Reply:
i think you need to push?